tests/aws/services/events/test_events_cross_account_region.py

```diff
@@ -320,3 +318,138 @@ def test_event_bus_to_event_bus_cross_account_region(
+        secondary_aws_client = response["secondary_aws_client"]
+        secondary_region_name = response["secondary_region_name"]
+        secondary_account_id = response["secondary_account_id"]
+
+        # overwriting randomized region https://docs.localstack.cloud/contributing/multi-account-region-testing/
+        # requires manually adding region replacement for snapshot
+        snapshot.add_transformer(snapshot.transform.regex(region_name, "<region>"))
+        snapshot.add_transformer(
+            snapshot.transform.regex(secondary_region_name, "<region-secondary>")
+        )
+        snapshot.add_transformer(snapshot.transform.regex(account_id, "<account>"))
+        snapshot.add_transformer(
+            snapshot.transform.regex(secondary_account_id, "<account-secondary>")
+        )
+
+        # Create event bus in secondary region / account
+        if event_bus_type == "default":
+            event_bus_name = "default"
+            event_bus_arn = f"arn:aws:events:{region_name}:{account_id}:event-bus/default"
+        else:
+            event_bus_name = f"test-bus-{short_uid()}"
+            response = secondary_aws_client.events.create_event_bus(Name=event_bus_name)
+            event_bus_arn = response["EventBusArn"]
+
+        # Allow principal to put event to event bus
+        secondary_aws_client.events.put_permission(
+            StatementId=f"TargetEventBusAccessPermission{short_uid()}",
+            EventBusName=event_bus_name,
+            Action="events:PutEvents",
+            Principal="*",
+        )
+
+        # Create SQS queue in secondary region / account
+        queue_name = f"test-queue-{short_uid()}"
+        queue_url = secondary_aws_client.sqs.create_queue(QueueName=queue_name)["QueueUrl"]
+        queue_arn = secondary_aws_client.sqs.get_queue_attributes(
+            QueueUrl=queue_url, AttributeNames=["QueueArn"]
+        )["Attributes"]["QueueArn"]
+        policy = {
+            "Version": "2012-10-17",
+            "Id": f"sqs-eventbridge-{short_uid()}",
+            "Statement": [
+                {
+                    "Sid": f"SendMessage-{short_uid()}",
+                    "Effect": "Allow",
+                    "Principal": {"Service": "events.amazonaws.com"},
+                    "Action": "sqs:SendMessage",
+                    "Resource": queue_arn,
+                }
+            ],
+        }
+        secondary_aws_client.sqs.set_queue_attributes(
+            QueueUrl=queue_url, Attributes={"Policy": json.dumps(policy)}
+        )
+
+        # Create rule in secondary region / account
+        rule_name = f"test-rule-sqs-{short_uid()}"
+        secondary_aws_client.events.put_rule(
+            Name=rule_name,
+            EventPattern=json.dumps({"source": [SOURCE_PRIMARY]}),
+            EventBusName=event_bus_name,
+        )
+
+        # Create target in secondary region / account
+        target_id_sqs_primary = f"test-target-primary-sqs-{short_uid()}"
```

Comment:
The variable name is misleading because it contradicts the comment: should it be secondary here?